Ability for Compressed PositionPackets to be Generated

chucklin85 opened the pull request:
Currently javAPRSlib cannot output compressed position packets, which is problematic for situations when APRSdroid or similar applications are transmitting using HF. Uncompressed location packets are generally frowned upon when used with APRS over HF

ge0rg commented:
Thanks for the contribution! The code looks reasonable to me and I'll gladly merge it. However, it would be very nice if you could squash together the two commits, so that there is no intermediate state that doesn't compile.

Thanks!

ge0rg commented:
P.S: I can do the squashing for you if you prefer so :)

ge0rg commented:
Oh, another minor nitpick: Setters and Getters in Java are starting with lowercase, and you might want to add a getCompressedFormat() method as well.

chucklin85 commented:
No problem! I will revise and resubmit 😃

chucklin85 commented:
Coming from a background of SVN, .NET development and primarily doing development on embedded equipment & systems it has been a very steep 24 hours! 
I've squashed the commit and fixed up setter name and added a getter :-)

ab0oo commented:
Yeah, my first thought was "Oh Geez, someone's USING it!"  (The
javAPRSlib, not git).   I STILL don't get git, 5 years on.  I'm an SVN
guy myself.  These kids and their new technology.

Let me know if you need anything documented or 'splained.  I can
probably make something up.

John


On 3/8/17 4:20 PM, chucklin85 wrote:
> Coming from a background of SVN, .NET development and primarily doing
> development on embedded equipment & systems it has been a very steep 24
> hours!
> I've squashed the commit and fixed up setter name and added a getter :-)
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ab0oo/javAPRSlib/pull/3#issuecomment-285202518>, or
> mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAXzkaIDO6Z07mARlYYpqZ61_Hm5Rb7zks5rjzepgaJpZM4MWdNB>.
>

chucklin85 commented:
javAPRSlib, it ain't broke so don't change it :-) Great library!
Latest commit is to allow for compressed packets to include altitude and course etc.
Squashed changes down with one more enhancement to permit setting the csT fields

ge0rg commented:
Merged and pushed, also added another fixup from my own codebase :)
